docs/source/how-to/custom-403.md

```diff
@@ -0,0 +1,27 @@
+# Custom login errors
+
+When a user is recognized by an identity provider but isn't allowed to sign in,
+the following message is shown by default.
+
+```
+Sorry, you are not currently authorized to use this hub. Please contact the hub administrator.
+```
+
+You can configure a custom error message like this:
+
+```python
+c.OAuthenticator.custom_403_message = "Your message for the user"
+```
+
+You can also show a customized 403 HTML page by creating a [custom HTML
+template], and configuring JupyterHub to find it.
```

Comment:
Or is this supposed to be pointing to a link?
```suggestion
You can also show a customized 403 HTML page by creating a custom HTML
template, and configuring JupyterHub to find it.
```

Reply:
I just realized you put the links at the end. Hmm ,I think it's nice they're separated. I like it!